Sources/SpeziLLMLocal/LLMLocalSession+Generate.swift

```diff
@@ -84,7 +83,15 @@ extension LLMLocalSession {
                 return .more
             }
             
-            return (result, tokenizer)
+            if schema.injectIntoContext {
+                // Yielding every Nth token may result in missing the final tokens.
+                let reaminingTokens = result.tokens.count % schema.parameters.displayEveryNTokens
+                let lastTokens = Array(result.tokens.suffix(reaminingTokens))
+                let text = tokenizer.decode(tokens: lastTokens)
+                continuation.yield(text)
```

Comment:
Just to clarify: This yield should just write the inferred output to the context, not yield it to the public API surface. That's what the `schema.injectIntoContext` was intended to do. It seems like we deviated a bit from that paradigm?

Reply:
Oh than I might have understood it in the wrong way, sorry about that. So in the current implementation `schema.injectIntoContext` will output the processed tokens ASAP while if its set to `false` it will return the whole assistants response as one.
Maybe we can find a better name for that, or implement the original idea if still needed.

Reply:
I would definitely keep the `schema.injectIntoContext` functionality as it was intended (automatically inject output of the model into the stored context without the user needing to ingest it manually; feel free to come up with a better name here, for me it always was quite clear :D), should also be documented in DocC and used by some applications that depend on SpeziLLM.
The "whole assistant response as one" can be easily achieved by writing an extension on the `AsyncStream` to fetch the entire `String` output, so something like:


```swift
extension AsyncStream where Element == String {
    func collectToString() async -> String {
        var result = ""
        for await chunk in self {
            result.append(chunk)
        }
        return result
    }
}
```

Therefore, I would discourage using the `schema.injectIntoContext` for that.